earthly/rust/Earthfile

```diff
@@ -1,5 +1,6 @@
 # Common Rust UDCs and Builders.
-VERSION --global-cache --use-function-keyword 0.7
+VERSION --use-function-keyword 0.7
+#VERSION --global-cache --use-function-keyword 0.7
```

Comment:
```suggestion
```

Reply:
This was to try and get the failing CI because (maybe) of the buildkit version mismatch to work.  It didn;t help so reverted it.